packages/amplify-migration-tests/package.json

```diff
@@ -17,6 +17,7 @@
   ],
   "private": true,
   "scripts": {
+    "build-tests": "tsc --build tsconfig.tests.json",
```

Comment:
this should be `yarn tsc` to make sure not a globally installed tsc will be used.